docs/blog.md

```diff
@@ -1,5 +1,12 @@
 # Microsoft MakeCode Blog
 
+## [Chibitronics Editor v0.1.40 (Scratch blocks)](/blog/chibitronics/v0.1.40)
+
+October 8th, 2017
+
+A big release for our Chibitronics editor!
+**[Read more](/blog/chibitronics/v0.1.40)**
```

Comment:
Should be 0.2 since you had API changes?

Reply:
Last released version was 0.0.60 so were good